Model/Product/Builder.php

```diff
@@ -240,6 +240,50 @@ public function build(
         return $nostoProduct;
     }
 
+    /**
+     * Adds selected attributes to all tags also in the custom fields section
+     *
+     * @param Product $product
+     * @param Store $store
+     * @return array
+     */
+    private function amendAttributesCustomFields(Product $product, Store $store)
+    {
+        $customFields = $this->buildCustomFields($product, $store);
+        $attributes = $this->getAttributesFromAllTags($store);
+        foreach ($product->getAttributes() as $key => $productAttribute) {
+            if (in_array($key, $attributes, false)) {
+                $attributeValue = $this->getAttributeValue($product, $key);
+                if (empty($attributeValue)) {
```

Comment:
It should be included if the value is 0. But empty() return true if the value is 0.